src/debugpy/_vendored/pydevd/pydevd_attach_to_process/windows/compile_windows.bat

```diff
@@ -18,7 +18,7 @@ cl -DUNICODE -D_UNICODE /EHsc /Zi /O1 /W3 /LD /MD /D BITS_32 /Qspectre run_code_
 copy run_code_on_dllmain_x86.dll ..\run_code_on_dllmain_x86.dll /Y
 copy run_code_on_dllmain_x86.pdb ..\run_code_on_dllmain_x86.pdb /Y
 
-cl /EHsc /Zi /O1 /W3 /Qspectre inject_dll.cpp /link /PROFILE /GUARD:CF /out:inject_dll_x86.exe
+cl /EHsc /Zi /O1 /W3 /Qspectre inject_dll.cpp /link /PROFILE /GUARD:CF /CETCOMPAT /out:inject_dll_x86.exe
```

Comment:
Will we have to fix these compiler and linker flags up every time we pull from PyDev.Debugger, or are we trying to push these changes back to him?

Reply:
I'm going to push this back to him now

Reply:
I had made these changes there but missed a spot, this is my changes coming back